.drone.yml

```diff
@@ -0,0 +1,26 @@
+---
+kind: pipeline
+name: linux_aarch64_
```

Comment:
Nevermind. Ignore my previous comment. Missed that `aarch64` was being moved to Drone (native build as opposed to an emulated one). Should be fine.